spec/spec.yml

```diff
@@ -5764,6 +5764,10 @@ definitions:
         extras:
           type: object
           description: 'Optional additional json metadata'
+        defaultLayerId:
+          type: string
+          format: uuid
+          description: 'Default project layer id for scenes to add to'
```

Comment:
I think "Default project layer to interact with" is a bit more accurate, since this isn't just about scene interaction.